README.md

```diff
@@ -18,23 +17,67 @@ Additional code examples can be accessed from the following locations:
+[Create a Feature Discovery project](https://docs.datarobot.com/en/docs/data/transform-data/feature-discovery/fd-overview.html) - Feature Discovery is based on relationships—between datasets and the features within those datasets.  \
+[Prediction intake options](https://docs.datarobot.com/en/docs/predictions/batch/batch-prediction-api/intake-options.html) - You can configure a prediction source using the Predictions > Job Definitions tab or the Batch Prediction API. \
+[Batch Prediction API : Snowflake scoring](https://docs.datarobot.com/en/docs/predictions/batch/batch-prediction-api/intake-options.html#snowflake-scoring) - Using JDBC to transfer data can be costly in terms of IOPS (input/output operations per second) and expense for data warehouses. This adapter reduces the load on database engines during prediction scoring by using cloud storage and bulk insert to create a hybrid JDBC-cloud storage solution. 
 
-# Code of Conduct
+### Data Processing Related
+[DataRobot Pipelines v7.3.0+](https://docs.datarobot.com/en/docs/data/pipelines/index.html) - DataRobot Pipelines enable data science and engineering teams to build and run machine learning data flows. Teams start by collecting data from various sources, cleaning them, and combining them.  
+
+### MLOps Custom Model Hosting on DataRobot
+[Workshop: Create, test, and deploy a custom model](https://docs.datarobot.com/en/docs/mlops/deployment/custom-models/index.html) - Custom inference models allow you to bring your own pretrained models to DataRobot. By uploading a model artifact, you can create, test, and deploy custom inference models to a centralized deployment hub. DataRobot supports models built with a variety of coding languages, including Python, R, and Java. 
+
+### MLOps Overview, Automatic Retraining, Accuracy Monitoring, MLOps Agent, Agent Use Cases )
+[MLOps overview](https://docs.datarobot.com/en/docs/mlops/mlops-overview.html) - DataRobot MLOps provides a central hub to deploy, monitor, manage, and govern all your models in production, regardless of how they were created or when and where they were deployed.  \
+[Continuous AI: Set up automatic retraining](https://docs.datarobot.com/en/docs/mlops/manage-mlops/set-up-auto-retraining.html) - To maintain model performance after deployment without extensive manual work, DataRobot provides an automatic retraining capability for deployments. \
+[Enable accuracy monitoring](https://docs.datarobot.com/en/docs/mlops/manage-mlops/setup-accuracy.html) - You can monitor a deployment for accuracy using the Accuracy tab, which lets you analyze the performance of the model deployment over time, using standard statistical measures and exportable visualizations. \
+[MLOps agent](https://docs.datarobot.com/en/docs/mlops/deployment/mlops-agent/index.html) - DataRobot MLOps provides powerful tools for tracking and managing models for prediction. But what if you already have—or need to have—deployments running in your own environment?  \
```

Comment:
do you want me to review at this level? @nate-goudreault ? because I am - hope it's what you need
I think you want another sentence after this question OR have this this question first then the answer in the 2nd sentence